include/serial/serial.h

```diff
@@ -67,20 +80,20 @@ Serial * get_serial(serial_id_t port);
 void configure_serial(serial_id_t port, uint8_t bits, uint8_t parity,
                       uint8_t stopBits, uint32_t baud);
 
-int serial_get_c_wait(const Serial *s, char *c, const size_t delay);
```

Comment:
Because I had to put in some state trackers into the Serial object, thus making Serial mutable.  Hence why the removal of the const.